### PR TITLE
Use same logging rules as ActiveRecord

### PR DESCRIPTION
**Pull request by ankane**
Hey, thanks for this awesome gem :+1:

This commit reduces logging noise by using [the same rules as ActiveRecord uses](https://github.com/rails/rails/blob/b06f64c3480cd389d14618540d62da4978918af0/activerecord/lib/active_record/log_subscriber.rb#L33).

### CODE REVIEW

File: lib/makara/logging/subscriber.rb [hunk 1]
```diff
@@ -2,12 +2,17 @@ module Makara
...
     module Subscriber
+      IGNORE_PAYLOAD_NAMES = ["SCHEMA", "EXPLAIN"]
 
       def sql(event)
         name = event.payload[:name]
-        name = [current_wrapper_name(event), name].compact.join(' ')
-        event.payload[:name] = name
-        super(event)
+        if IGNORE_PAYLOAD_NAMES.include?(name)
+          self.class.runtime += event.duration
```
Comment (mnelson) [inline lib/makara/logging/subscriber.rb]: Wouldn't you just want to call super here?

File: lib/makara/logging/subscriber.rb [hunk 2]
```diff
@@ -2,12 +2,17 @@ module Makara
...
 
     module Subscriber
+      IGNORE_PAYLOAD_NAMES = ["SCHEMA", "EXPLAIN"]
 
       def sql(event)
         name = event.payload[:name]
-        name = [current_wrapper_name(event), name].compact.join(' ')
-        event.payload[:name] = name
-        super(event)
+        if IGNORE_PAYLOAD_NAMES.include?(name)
```
Comment (mnelson) [inline lib/makara/logging/subscriber.rb]: This should just use `ActiveRecord::LogSubscriber::IGNORE_PAYLOAD_NAMES` directly if possible. Are there AR version semantics?

Comment (ankane) [inline lib/makara/logging/subscriber.rb]: That's what I originally did, but `IGNORE_PAYLOAD_NAMES` isn't present before ActiveRecord 4 and could disappear at any time, so it's a bit safer to define it ourselves.  I don't feel too strongly either way.

File: lib/makara/logging/subscriber.rb (hunk 1 above)
Comment (ankane) [inline lib/makara/logging/subscriber.rb]: Good call.

**Pull request merged**